ghost/core/core/shared/config/defaults.json

```diff
@@ -102,6 +102,12 @@
             "maxWait": 43200000,
             "lifetime": 43200,
             "freeRetries": 8
+        },
+        "mentions_block": {
+            "minWait": 60,
+            "maxWait": 600,
+            "lifetime": 6000,
+            "freeRetries": 2
```

Comment:
For the values, maybe let's try and use some values that will have little to no impact initially?

I haven't as much context as you but looking at https://github.com/AdamPflug/express-brute/blob/master/index.js maybe something like:
```
{
  minWait: 1,
  maxWait: 1,
  lifetime: 0,
  freeRetries: 100
}
```

Reply:
Couldn't get the values that liberal as then tests cannot reproduce a block - simply resets too fast.
Made it a little stricter than your suggestion, but still way more liberal compared to what it was.

```
"webmentions_block": {
            "minWait": 10,
            "maxWait": 100,
            "lifetime": 1000,
            "freeRetries": 100
        }
```